tests/repl/vars.check

```diff
@@ -1,8 +1,8 @@
 scala> var x = 0
-x: Int = 0
+var x: Int = 0
 scala> x = x + 1
 x: Int = 1
 scala> x *= 2
 scala> x
-res2: Int = 2
+val res2: Int = 2
```

Comment:
Indeed, the REPL currently has semantics that are not well defined. There's no spec for it :sweat_smile: 

But, if you want to have a closer look at its semantics, by all means! We'd be happy to get something that is clearer :)